recipes/libjpeg-turbo/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "1.5.1" %}
+{% set libjpeg_version = "8c" %}
+
+package:
+    name: libjpeg-turbo
+    version: {{ libjpeg_version }}_{{ version }}
```

Comment:
Almost. `libjpeg-turbo` is a replacement for `libjpeg` (the `jpeg` package) and can only be compatible with one major version of `libjpeg` at a time. The packages that are going to depend on it will care much more about the `libjpeg` interface than the `libjpeg-turbo` version, especially since all `libjpeg-turbo` versions are backwards-compatible. So it seemed important enough to include in the package version.

Having said that, leaving out the `libjpeg` version would not be a bad choice actually. All packages available in conda depend on `libjpeg` version 8 or 9 and the developer of `libjpeg-turbo` [has decided to finally split off `libjpeg` since version 8](http://www.libjpeg-turbo.org/About/Jpeg-9). So the `libjpeg` version part can be reasonably expected to stay the same in the future. Considering this, I'd day we remove it.